vllm/model_executor/models/utils.py

```diff
@@ -506,6 +507,12 @@ def maybe_offload_to_cpu(module: torch.nn.Module) -> torch.nn.Module:
 
     pin_memory = is_pin_memory_available()
 
+    if envs.VLLM_USE_V1:
+        assert pin_memory, "V1 CPU offloading requires pin_memory support"
```

Comment:
nit: it is better to use is_uva_available() here for future-proofing, although the assert message will be less clear

Reply:
good point, fixed in [1bb216c](https://github.com/vllm-project/vllm/pull/15354/commits/1bb216c6d3714f20f1c0748caab6304c6b8f0ac5)